SECURITY.md

```diff
@@ -0,0 +1,63 @@
+# Security Policy
+
+This document describes the Security Policy for ApeWorX Ltd., including vulnerability disclosures for the [Ape Framework](https://github.com/ApeWorX/ape) and any of it's [maintained plugins](https://github.com/search?q=org%3Aapeworx+ape-+-repo%3AApeWorX%2Fape++-repo%3AApeWorX%2Fapeworx.github.io&type=Repositories).
+We are committed to conduct our Security Process in a professional and civil manner.
+Public shaming, under-reporting, or misrepresentation of vulnerabilities will not be tolerated.
```

Comment:
Are there consequences? What does this mean?

Reply:
I don't know, might be more specific to smart contract exploits